src/main/java/io/fabric8/jenkins/openshiftsync/GlobalPluginConfiguration.java

```diff
@@ -59,13 +61,14 @@
   private transient ConfigMapWatcher configMapWatcher;
 
   @DataBoundConstructor
-  public GlobalPluginConfiguration(boolean enable, String server, String namespace, String jobNamePattern, String skipOrganisationPrefix, String skipBranchSuffix) {
+  public GlobalPluginConfiguration(boolean enable, String server, String namespace, String jobNamePattern, String skipOrganisationPrefix, String skipBranchSuffix, boolean cancelPRbuildOnUpdate) {
     this.enabled = enable;
     this.server = server;
     this.namespace = namespace;
     this.jobNamePattern = jobNamePattern;
     this.skipOrganisationPrefix = skipOrganisationPrefix;
     this.skipBranchSuffix = skipBranchSuffix;
+    this.cancelPRBuildOnUpdate = cancelPRbuildOnUpdate;
```

Comment:
Shouldn't there be a config panel change to go along with this addition to the config.

Reply:
Can you guide me how to do that, no idea and experience with this.

We are setting these value using configMap and it's getting reflected.

Reply:
good job - you found it below  ... will comment further there

Reply:
@gabemontero you mean updating its jelly file?